src/objects/zcl_abapgit_sotr_handler.clas.abap

```diff
@@ -0,0 +1,199 @@
+class ZCL_ABAPGIT_SOTR_HANDLER definition
+  public
+  final
+  create public .
+
+  public section.
+    types: yt_SOTR_USE  type standard table of SOTR_USE with default key,
+           yt_seocompodf TYPE STANDARD TABLE OF seocompodf WITH DEFAULT KEY.
+
+  class-methods read_sotr_wda
+    importing
+      iv_object_name TYPE sobj_name
+    RETURNING
+      VALUE(rt_sotr) TYPE zif_abapgit_definitions=>ty_sotr_tt
+    RAISING
+      zcx_abapgit_exception.
+  class-methods read_sotr_seocomp
+    IMPORTING
+      iv_object_name TYPE sobj_name
+    RETURNING
+      VALUE(rt_sotr) TYPE zif_abapgit_definitions=>ty_sotr_tt
+    RAISING
+      zcx_abapgit_exception.
+  class-methods create_sotr
+    IMPORTING
+      iv_package TYPE devclass
+      it_sotr    TYPE zif_abapgit_definitions=>ty_sotr_tt
+    RAISING
+      zcx_abapgit_exception.
+
+   PROTECTED SECTION.
+     class-methods get_sotr_4_concept
+       importing
```

Comment:
@larshp can you please explain this check for us?

Reply:
its related to whitespace

Currently there are 5 spaces before "class-methods"
```
     class-methods get_sotr_4_concept
```

indentation in abap is typically done using tabs, which are 2 spaces, ie. all indentation should be at tab positions(spaces MOD 2 = 0)

solution: add another space at start of line

Reply:
similar with 

```   PROTECTED SECTION.```

which have 3 spaces

Reply:
actually just run the SAP standard pretty printer on the source code, with upper case keywords + indentation